### PR TITLE
Foreground color updates header and icon

### PR DESCRIPTION
**Pull request by jherbert**
Commit 4cf183810f726d1b4da46023adf7a01d462bddc2 updated expander to use PackIcon, pull request so foreground is reflected both on header text and icon.

### CODE REVIEW

File: MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Expander.xaml [hunk 1]
```diff
@@ -70,7 +70,7 @@
                             </VisualStateGroup>
                         </VisualStateManager.VisualStateGroups>
                         <wpf:PackIcon Kind="ChevronDown" Opacity=".38" x:Name="ExpandPath" RenderTransformOrigin="0.5,0.5"
-                                      Height="24" Width="24">
+                                      Height="24" Width="24" Foreground="{Binding RelativeSource={RelativeSource AncestorType={x:Type Expander}}, Path=Foreground}">
```
Comment (ButchersBoy) [inline MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Expander.xaml]: Is a TemplateBinding not sufficient here?

**Comment by jherbert** [comment]
Updated to use `TemplateBinding` for the foreground on each style.

md_expander_ss

**Comment by ButchersBoy** [comment]
Hi, please see my code review comment, I don't think we need the relative source binding as you are setting the toggle foreground, and the binding is in the toggle template.

File: MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Expander.xaml (hunk 1 above)
Comment (ButchersBoy) [inline MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Expander.xaml]: @jherbert

**Comment by jherbert** [comment]
Sorry you are correct. Had it that way locally but failed to push it up.

**Comment by ButchersBoy** [comment]
Thankyou!

**Pull request merged**